lints/lint_mp_allowed_rsa_keys_test.go

```diff
@@ -0,0 +1,54 @@
+package lints
+
+/*
+ * ZLint Copyright 2018 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestSerialEntropyBits(t *testing.T) {
```

Comment:
I think the name of this test function doesn't match what it's actually testing.

Reply:
Please see Zakir's comment on this, I am going to correct it.